README.md

```diff
@@ -1,7 +1,10 @@
 ## Introduction
-This repository contains Kyber utils smart contracts.
+This repository contains Kyber utils smart contracts.\
```

Comment:
why do we need the \ at end of the line?

Reply:
It's a markdown file thing to break the line to the next line. Otherwise, if you just press enter in the .md file, there won't be a line break when it's published and will appear as just 1 straight line.

Reply:
Got it thanks